components/support/nimbus-fml/src/workflows.rs

```diff
@@ -158,6 +160,37 @@ pub(crate) fn fetch_file(files: &LoaderConfig, nm: &str) -> Result<()> {
     Ok(())
 }
 
+pub(crate) fn validate(cmd: &ValidateCmd) -> Result<()> {
+    let files: FileLoader = TryFrom::try_from(&cmd.loader)?;
+
+    let filename = &cmd.manifest;
+    let file_path = files.file_path(filename)?;
+    let parser: Parser = Parser::new(files, file_path.clone())?;
+    let mut loading = HashSet::new();
+    let fe = parser.load_manifest(&file_path, &mut loading)?;
```

Comment:
fe?

Reply:
We have a habit of short-naming things based off their type — `fe` is `ManifestFrontEnd`, `th` you'll see around for `NimbusTargetingHelper`, etc... 

I can swap it to be more descriptive though